server/routes/user.route.js

```diff
@@ -23,6 +25,13 @@ router.route('/:userId')
     /** DELETE /api/users/:userId - Delete user */
     .delete(userCtrl.remove);
 
+router.route('/scopes/:userId')
+    /** PUT /api/users/scopes/:userId - Update user scopes */
```

Comment:
tiny thing, but this should be `/api/user/scopes/:userId` in the comment

Reply:
Good catch!